### PR TITLE
Add the initial bootstrap governance proposal.

### PR DESCRIPTION
**Pull request by dlorenc**
@kimsterv @bobcatfish @abayer @vdemeester 

I'd love comments from everyone interested, but will wait for LGTM's from each member before merging.

### CODE REVIEW

File: governance.md [hunk 1]
```diff
@@ -1,14 +1,88 @@
...
-Establishing governance is still a work in progress and for now, Tekton is
-overseen by the Knative Build WG.
+The initial bootstrap committee will consist of 5 individuals and one governance facilitator who are core stakeholders and/or contributors.
 
-## Procedures
+Members are:
 
-### Creating a new repository
+* [Andrew Bayer](https://github.com/abayer) (CloudBees)
+* [Vincent Demeester](https://github.com/vdemeester) (RedHat)
```
Comment (vdemeester) [inline governance.md]: (absolute) nit: `s/RedHat/Red Hat/g`

Comment (dlorenc) [reply governance.md]: Haha yeah, forgot the hold. I'll get that updated.

File: governance.md [hunk 2]
```diff
@@ -1,14 +1,88 @@
...
-maintainers (for now these can be found in [Github](https://github.com/orgs/tektoncd/teams/maintainers/members).
+The committee MUST:
+
+* Represent a cross-section of interests, not just one company
+* Balance technical, architectural, and governance expertise since its initial mission is the establishment of structure around contributions, community, and decision-making
+* Hold staggered terms, sufficient to ensure an orderly transition of power via elections as designed and implemented by the committee (see below for specific deliverables)
+* Provide designated alternates in cases where quorum is required but not attainable with the current set of members
+* Communicate with the Continuous Delivery Foundation on a regular cadence
+
+## Governance Facilitator Role (optional)
```
Comment (bobcatfish) [inline governance.md]: we don't have one of these atm tho right? so this is just describing it in case we use it later?

File: governance.md [hunk 3]
```diff
@@ -1,14 +1,88 @@
...
+  * Defines how release decisions are made
+    * Who is authorized to release and when.
+    * Frequency limits
+  * Defines the documentation process
+  * Defines what CLA process is required and how it is programmatically enforced before code is merged
+
+## Code of Conduct
+
+The code of conduct MUST set expectations for contributors on expected behavior, as well as explaining the consequences of violating the terms of the code.
+The Contributor Covenant has become the de facto standard for this language.
```
Comment (bobcatfish) [inline governance.md]: should probably link here - we can link to our existing code of conduct docs as well?

File: governance.md [hunk 4]
```diff
@@ -1,14 +1,88 @@
...
+
+This document will define how the committee is to manage the project until it has transitioned to an elected steering body, as well as what governance must be in place.
+The Kubernetes Steering Committee Charter Draft serves as a good example. 
+
+A charter should cover all of the following topics:
+
+* Scope of rights and responsibilities explicitly held by the committee
+* Committee structure that meets the requirements above
+* Election process, including: 
+  * special elections in the case someone resigns or is impeached
```
Comment (bobcatfish) [inline governance.md]: in all seriousness i cant stop myself from asking: how does a person get impeached O.O

Comment (dlorenc) [reply governance.md]: This is a list of things this steering committee is required to outline/determine. So, we get to decide!

File: governance.md (hunk 2 above)
Comment (dlorenc) [reply governance.md]: Correct, we need to decide whether to use this role.

**Pull request merged**